plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -250,6 +250,27 @@ class read_only {
 
    get_table_rows_result get_table_rows( const get_table_rows_params& params )const;
 
+   struct get_table_by_scope_params {
+      name        code; // mandatory
+      name        table = 0; // optional, act as filter
+      string      lower_bound; // lower bound of scope, optional
+      string      upper_bound; // upper bound of scope, optional
+      uint32_t    limit = 10;
+   };
+   struct get_table_by_scope_result_row {
+      name        code;
+      name        scope;
+      name        table;
+      name        payer;
+      uint32_t    count;
+   };
+   struct get_table_by_scope_result {
+      vector<fc::variant> rows; ///< one row per item, either encoded as hex String or JSON object
```

Comment:
Why is this a variant? Also the comment seems wrong.

Reply:
Will fix this accordingly.